sdk/communication/AzureCommunicationChat/Source/ChatClient.swift

```diff
@@ -330,4 +333,154 @@ public class ChatClient {
 
         signalingClient!.off(event: event)
     }
+
+    /// Start push notifications. Receiving of notifications can be expected ~1 second after successfully registering.
+    /// - Parameters:
+    ///   - deviceToken: APNS push token.
+    ///   - completionHandler: Success indicates request to register for notifications has been received.
+    public func startPushNotifications(
+        deviceToken: String,
+        encryptionKeys: [String: String],
```

Comment:
The SDK should create handle the keys transparently. The values are presumably the encryption keys, but what are the keys to this dictionary?

Reply:
This dictionary follows the below format -
{
    "FIRST_KEY": "XXXXXX",
    "SECOND_KEY": "YYYYYY"
}

We will need to guide Contoso to put "FIRST_KEY" and "SECOND_KEY" as the keys to the dictionary.
Let me know if you think it is ok

Reply:
No, that's definitely not okay. If that is the structure of the dictionary then this shouldn't be a dictionary at all. It should be an object with `firstKey` and `secondKey` string properties. 

More fundamentally, these should not be exposed.

Reply:
Unless there's a compelling reason to be a class, it should almost certainly be a struct.